### PR TITLE
retrieve tax rate from the back-end

### PR DESCRIPTION
**Pull request by mauro-fanton**
### Description of Work carried through

Brief description of what/why/how.

### CODE REVIEW

**Review by muninderadavelly — APPROVED** [review]

**Review by lmcewanhmrc — APPROVED** [review]

File: app/viewmodels/checkAnswers/ViewReturnSummaryViewModel.scala [hunk 1]
```diff
@@ -59,34 +59,34 @@ final case class ViewReturnSummaryViewModel(summarySection : Section, detailsSec
 
 object ViewReturnSummaryViewModel {
 
-  private def toLocalDate(date: String) = LocalDate.parse(date, DateTimeFormatter.ISO_DATE_TIME)
+  private def   toLocalDate(date: String) = LocalDate.parse(date, DateTimeFormatter.ISO_DATE_TIME)
```
Comment (liampan) [inline app/viewmodels/checkAnswers/ViewReturnSummaryViewModel.scala:62]:
```suggestion
  private def toLocalDate(date: String) = LocalDate.parse(date, DateTimeFormatter.ISO_DATE_TIME)
```

File: app/views/amends/ViewReturnSummaryView.scala.html [hunk 2]
```diff
@@ -30,7 +30,7 @@
     paragraphBody: paragraphBody
 )
 
-@(returnPeriod: String, vm: ViewReturnSummaryViewModel, amendCall: Option[Call])(implicit request: Request[_], messages: Messages)
+@(returnPeriod: String, vm: ViewReturnSummaryViewModel, amendCall: Option[Call], taxRateInPoundPerTonne: String)(implicit request: Request[_], messages: Messages)
```
Comment (liampan) [inline app/views/amends/ViewReturnSummaryView.scala.html:33]: is this not in the vm?

Comment (liampan) [reply app/views/amends/ViewReturnSummaryView.scala.html:33]: or can the view not call the viewmodels implicit method

File: test/viewmodels/ViewmodelsSpec.scala [hunk 3]
```diff
@@ -62,4 +63,26 @@ class ViewmodelsSpec extends PlaySpec {
     }
   }
 
+  "asPoundPerTonne" should {
```
Comment (liampan) [inline test/viewmodels/ViewmodelsSpec.scala:66]: nice :)

File: test/views/amends/CheckYourAnswersViewSpec.scala [hunk 4]
```diff
@@ -117,8 +117,8 @@ class CheckYourAnswersViewSpec extends ViewSpecBase with ViewAssertions with Vie
 
   private def createCalculations(isSubmittable: Boolean) = {
     AmendsCalculations(
-      Calculations(12, 40, 100, 200, isSubmittable),
-      Calculations(12, 40, 100, 200, isSubmittable)
+      Calculations(12, 40, 100, 200, isSubmittable, 200.0),
```
Comment (liampan) [inline test/views/amends/CheckYourAnswersViewSpec.scala:120]: 200 or 0.2 🤔

Comment (liampan) [reply test/views/amends/CheckYourAnswersViewSpec.scala:120]: not that it matters

**Review by liampan — APPROVED** [review]
Maybe somethings we can tweak going forward but lgtm :D 🥳

File: app/viewmodels/checkAnswers/ViewReturnSummaryViewModel.scala (hunk 1 above)
Comment (liampan) [reply app/viewmodels/checkAnswers/ViewReturnSummaryViewModel.scala:62]: @mauro-fanton just commit from here when merge :)

**Pull request merged**